.github/workflows/pre-merge-checks.yml

```diff
@@ -188,8 +251,6 @@ jobs:
         with:
           key: ${{ runner.os }}-integration-test-reports-${{ github.sha }}
           path: |
-            integration-tests/build/jacoco/
-            integration-tests/build/reports/
```

Comment:
Not your change, but I think there's a typo on line 257 of the updated code (line 196 on old) - there's an extra colon in `delivery--receipts-integration-tests/build/reports/`